backend/templates/index.html

```diff
@@ -9,6 +9,13 @@
 
       <!-- favicon -->
       <link rel="icon" href="/static/favicon.png">
+      <!-- Custom CSS -->
+      <style>
+      body {
```

Comment:
you should use this as an excuse to do a custom css file and then build that into your front-end project with webpack. it's not that hard, I can walk you through the basics if you need to.